Bio/Entrez/Parser.py

```diff
@@ -328,7 +328,7 @@ def read(self, handle):
         if hasattr(handle, "closed") and handle.closed:
             # Should avoid a possible Segmentation Fault, see:
             # http://bugs.python.org/issue4877
-            raise IOError("Can't parse a closed handle")
+            raise ValueError("Can't parse a closed handle")
```

Comment:
Bio.Entrez has already been fixed

Reply:
Only this file now changing among the Entrez files. `pyupgrade` leaves the other new files unchanged